src/lib/support/TestSceneTable.h

```diff
@@ -72,12 +73,27 @@ static uint8_t CC_buffer[scenes::kMaxFieldsPerCluster] = { 0 };
 class TestSceneHandler : public scenes::DefaultSceneHandlerImpl
 {
 public:
-    static constexpr uint8_t kMaxValueSize = 4;
-    static constexpr uint8_t kMaxAvPair    = 15;
-
     TestSceneHandler() = default;
     ~TestSceneHandler() override {}
 
+    // Fills in cluster buffer and adjusts its size to lower than the maximum number of cluster per scenes
+    virtual void GetSupportedClusters(EndpointId endpoint, Span<ClusterId> & clusterBuffer) override
+    {
+        ClusterId * buffer = clusterBuffer.data();
```

Comment:
Here as well: check the .size() before touching.

Reply:
Added check for size in latest commit.